### DISCUSSION TITLE
Metro Channel

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#670] [Q&A]
**Channel Name:** Metro Channel
**Channel ID:** MetroChannel.ph
**Country:** Philippines
**Language:** Tagalog/English
**External Link (optional):** xxx
**Notes (optional):** Lifestyle channel in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19668
